frontend/src/layout/Sidebar.js

```diff
@@ -54,51 +59,91 @@ export default function Sidebar(props) {
     const { scene, loadingScene } = useValues(sceneLogic)
     const { location } = useValues(router)
     const { push } = useActions(router)
+    const { dashboards, pinnedDashboards } = useValues(dashboardsModel)
 
-    const activeScene = sceneOverride[loadingScene || scene] || loadingScene || scene
+    let activeScene = sceneOverride[loadingScene || scene] || loadingScene || scene
+    const openSubmenu = submenuOverride[activeScene] || activeScene
+
+    let firstDashboard = pinnedDashboards.length > 0 ? `/dashboard/${pinnedDashboards[0].id}` : '/dashboard'
+    let unpinnedDashboard = null
+    if (activeScene === 'dashboard') {
+        const dashboardId = parseInt(location.pathname.split('/dashboard/')[1])
+        activeScene = `dashboard-${dashboardId}`
+
+        const dashboard = dashboards.find(d => d.id === dashboardId)
+        if (dashboard && !dashboard.pinned) {
+            unpinnedDashboard = dashboard
+        }
+    }
 
     return (
-        <Layout.Sider breakpoint="lg" collapsedWidth="0" className="bg-light">
+        <Layout.Sider breakpoint="lg" collapsedWidth="0" className="bg-dark">
             <Menu
-                className="h-100 bg-light"
+                className="h-100 bg-dark"
+                theme="dark"
                 selectedKeys={[activeScene]}
-                openKeys={[submenuOverride[activeScene] || activeScene]}
+                openKeys={[openSubmenu]}
                 mode="inline"
             >
                 <Logo />
                 <Menu.Item key="trends" style={itemStyle}>
                     <RiseOutlined />
-                    <span>{'Trends'}</span>
+                    <span className="sidebar-label">{'Trends'}</span>
                     <Link to={'/trends'} />
                 </Menu.Item>
-                <Menu.Item key="dashboard" style={itemStyle}>
-                    <HomeOutlined />
-                    <span>{'Dashboard'}</span>
-                    <Link to={'/dashboard'} />
-                </Menu.Item>
+                <Menu.SubMenu
+                    key="dashboards"
+                    title={
+                        <span style={itemStyle}>
+                            <HomeOutlined />
+                            <span className="sidebar-label">Dashboards</span>
+                        </span>
+                    }
+                    onTitleClick={() => (location.pathname !== firstDashboard ? push(firstDashboard) : null)}
+                >
+                    {pinnedDashboards.map(dashboard => (
```

Comment:
My idea was to have these pinned dashboards open at all times so it's super easy to go to them

Reply:
Also in my original design (#703) I had it slightly separated from the rest for that reason. Happy with pushback on this.

Reply:
I tried it and I'm not sure it's better. Now that "dashboards" opens the list on "all dashboards" and is pretty fast, clicking "dashboards" and then directly clicking the pinned "my dash" after seems rather fast.

Keeping this menu always open and on top looks a bit off:

![image](https://user-images.githubusercontent.com/53387/81574897-ac223a80-93a6-11ea-9107-c5f491b15b1c.png)

I could remove "all dashboards", but then there's nothing to highlight (submenu titles can't be highlighted)

I'd keep it as and try using it like this for a week. We can always change the behaviour after if the closed menu feels off.

Reply:
Sounds good